packages/storybook/.storybook/blocks/story-layout.css

```diff
@@ -1,5 +1,3 @@
-@import url('@ni/nimble-tokens/dist/styledictionary/css/variables.css');
```

Comment:
saw in devtools that SB was importing base tokens but they don't seem to be used so removed.